goal_src/levels/village_common/oracle.gc

```diff
@@ -5,3 +5,811 @@
+            kill
+            "villageb-part-32"
+            )
+           (0
+            kill
+            "villageb-part-33"
+            )
+           (0
+            kill
+            "villageb-part-35"
+            )
+           (0
+            kill
+            "villageb-part-37"
+            )
+           (0
+            kill
+            "villageb-part-38"
+            )
+           (0
+            kill
+            "villageb-part-47"
+            )
+           (0
+            kill
+            "villageb-part-48"
+            )
+           (0
+            kill
+            "villageb-part-49"
+            )
+           (0
+            kill
+            "villageb-part-50"
+            )
+           (0
+            kill
+            "villageb-part-56"
+            )
+           (0
+            kill
+            "villageb-part-62"
+            )
+           (0
+            kill
+            "villageb-part-63"
+            )
+           (0
+            kill
+            "villageb-ogreboss-1"
+            )
+           (0
+            kill
+            "crate-3127"
+            )
+           (0
+            kill
+            "crate-3128"
+            )
+           (0
+            kill
+            "crate-3129"
+            )
+           (0
+            kill
+            "crate-3133"
+            )
+           (0
+            kill
+            "geologist-1"
+            )
+           (0
+            kill
+            "fireboulder-2"
+            )
+           (0
+            kill
+            "fireboulder-3"
+            )
+           (0
+            kill
+            "fireboulder-4"
+            )
+           (0
+            kill
+            "water-vol-100"
+            )
+           (0
+            kill
+            "water-vol-101"
+            )
+           (0
+            kill
+            "water-vol-102"
+            )
+           (0
+            kill
+            "water-vol-103"
+            )
+           (0
+            kill
+            "water-vol-284"
+            )
+           (0
+            kill
+            "water-vol-285"
+            )
+           (0
+            kill
+            "water-vol-286"
+            )
+           (0
+            kill
+            "ceilingflag-6"
+            )
+           (0
+            kill
+            "ceilingflag-7"
+            )
+           (0
+            kill
+            "ceilingflag-8"
+            )
+           (0
+            kill
+            "ceilingflag-9"
+            )
+           (0
+            kill
+            "orb-cache-top-14"
+            )
+           (0
+            setting-reset
+            ocean-off
+            near
+            )
+           ((the binteger 399) setting-unset ocean-off)
+           )
+         )
+        )
+       (('village3)
+        (new 'static 'spool-anim
+         :name "oracle-intro-3"
+         :index 13
+         :parts 4
+         :command-list '()
+         )
+        )
+       )
+      )
+    ((5)
+     (set! (-> obj skippable) #t)
```

Comment:
it's interesting that they set this for some cutscenes, but it seems like it's not actually skippable.